docs/examples/userguide/language_basics/exception_example.pyx

```diff
@@ -0,0 +1,113 @@
+# >>> c.a_cpdef_wo_py_retval()
+# Exception: a_cpdef_wo_py_retval: This Exception will cause a return of 0
```

Comment:
> This Exception will cause a return of 0

I don't know that this is something that users should rely on or attempt to test. From a user point-of-view it should probably be treated as "undefined" (even if it's pretty reliable in reality)

Reply:
Yes, it's definitely not something that should be relied upon, but it is a possible source of bugs (in users' programmes that is) if a function starts returning 0 because of an error, especially taking into account the previous language around "ignoring exceptions" which did not make the immediate return.